### PR TITLE
[Grid] Disabling actions + [Documentation][GridBundle][Customization] Update

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| Doc fix?        | yes |
| New feature?    | yes |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Bundle/UiBundle/Resources/views/Macro/table.html.twig [hunk 1]
```diff
@@ -26,14 +26,19 @@
         <td>
             <div class="ui buttons">
                 {% for action in definition.getActions('item')|sort_by('position') %}
-                    {{ sylius_grid_render_action(grid, action, row) }}
+                    {{ dump(action) }}
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/UiBundle/Resources/views/Macro/table.html.twig]: :)

Comment (TheMadeleine) [inline src/Sylius/Bundle/UiBundle/Resources/views/Macro/table.html.twig]: :D corrected.

**Comment by pjedrzejewski** [comment]
Thank you Magda!

File: docs/bundles/SyliusGridBundle/configuration.rst [hunk 2]
```diff
@@ -32,15 +30,38 @@ Here you will find all configuration options of ``sylius_grid``.
...
+                        position: 100
                         options: { }
                 actions:
                     main:
                         create:
                             type: create
+                            label: sylius.ui.create
+                            enabled: true
+                            icon: ~
+                            position: 100
```
Comment (michalmarcinkowski) [inline docs/bundles/SyliusGridBundle/configuration.rst]: Do we need to specify `position` and `enabled: true`? IMO these should be default values.

Comment (pjedrzejewski) [inline docs/bundles/SyliusGridBundle/configuration.rst]: This is reference.

**Pull request merged**